integration/cypress/integration/accessibility/accessibility.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { generateLegacyURL, generateNewURL } from "@maas-ui/maas-ui-shared";
+
+const legacyPages = ["/devices", "/controllers", "/networks"];
+
+type Page = { heading: string; url: string };
+const pages: Page[] = [
+  { heading: "Login", url: "/accounts/login" },
+  { heading: "SSH keys for admin", url: "/intro/user" },
```

Comment:
We should include the MAAS intro page at `/intro` with the heading "Welcome to MAAS" that is only accessible if the MAAS hasn't been set up for the first time (or when `shouldSkipIntro` is false

Reply:
Seems like `shouldSkipIntro = false` isn't enough, this would require running this test without any users (like we do in `no-users` directory). Doesn't seem worth the effort, I'd rather skip this for now if that's okay with you?